standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -752,6 +752,9 @@ public static ConfVars getMetaConf(String name) {
     EVENT_DB_LISTENER_CLEAN_INTERVAL("metastore.event.db.listener.clean.interval",
             "hive.metastore.event.db.listener.clean.interval", 7200, TimeUnit.SECONDS,
             "sleep interval between each run for cleanup of events from the database listener queue"),
+    EVENT_DB_LISTENER_CLEAN_STARTUP_WAIT_INTERVAL("metastore.event.db.listener.clean.startup.wait.interval",
+        "hive.metastore.event.db.listener.clean.startup.wait.interval", 1, TimeUnit.DAYS,
+        "wait interval post start of Cleaner thread after which the cleaner thread starts to work"),
```

Comment:
a) Capitalize "wait".
b) Is there a difference between Cleaner thread start time vs HMS process uptime(not considering time taken to launch a thread which idea would be very minimal)? If not, then referring to process start time would be better to convey the same.

Reply:
Done